### PR TITLE
Billing storage

### LINKED ISSUE #1969: [Storage] Enable Billing for Storage
Add the Billing module to Storage settings - only enabled for whitelisted users
Billing flows should be the same as those in Files.

* Do not include the account restricted mode in this change

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1969
I left out the notifications so that it doesn't interfere too much with the design refactor #1968

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode (there's no dark mode on Storage)

### CODE REVIEW

File: packages/storage-ui/src/Components/Pages/SettingsPage.tsx [hunk 1]
```diff
@@ -174,6 +178,16 @@ const SettingsPage: React.FC = () => {
             >
               <ApiKeys />
             </TabPane>
+            <TabPane
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Pages/SettingsPage.tsx]: This needs to be rendered conditionally with `{isBillingEnabled ? <TabXYX /> : null}`

**Comment by asnaith** [comment]
Just noticed a couple of issues

1) The link colour is the wrong shade of blue 
Screen Shot 2022-02-25 at 2 39 56 PM

Screen Shot 2022-02-25 at 2 50 32 PM

2) Add / update card modals are not working because the inputs are not getting the response from stripe to activate. 

I was able to test the upgrade and download flows successfully by adding a card to the profile via **Files** and then logging back into Storage. I had no issues upgrading with credit card or downgrading. I didn't pay via crypto but everything else with the flow and ability to return etc is all good.

**Comment by Tbaut** [comment]
Thanks Andrew for the review. I fixed the color issue for the links, and checked with the CC, from the Storage interface I was able to delete one (that I added from Files), and add one. Also note that you should make sure to have the Stripe pk env variable in you local environment. I just added to Render as it wasn't present. This should solve your issues.

Other than that things worked well for my tests.

**Review by tanmoyAtb — APPROVED** [review]
Great ! All the flows are working nicely ! 👍

**Comment by asnaith** [comment]
>  Also note that you should make sure to have the Stripe pk env variable in you local environment. I just added to Render as it wasn't present. This should solve your issues.

Ah yes, completely forgot about the .env file! Thanks, I'll try again with that in place

**Comment by asnaith** [comment]
I was able to test properly locally after adding the stripe key to the Storage UI .env 😅. 

The feature is working well, just two minor points remaining:

- The whitelist for the subscription tab access, at the moment all can access it
- The stripe key for render might need looking at (it still wasn't working for me)

**Comment by Tbaut** [comment]
I relaunched the deployment for render.. and checked, the env variable is definitely set :shrug: 

Pushing the whitelist check so that we don't forget it. Any reviewer can comment it.

**Review by asnaith — APPROVED** [review]
Yep, I can see the latest change has implemented the whitelist 👍

Render is still not working with stripe elements for me but approving as that has nothing to do with code in this PR.

**Comment by Tbaut** [comment]
I'll check the render issue more in details

**Pull request merged**